### PR TITLE
Add Stamps and Sprites

### PR DESCRIPTION
**Pull request by Lyacs**
## About the PR

This PR is aimed to add more lore friendly stamps to the game. 

In this PR I also introduce a new faction inspired by an old one. UWU, the united workers utopia. Throught it I wish a bit more diversity to the usual antagonists, and provide I explaination on how revolutionaries get their flash technology. (see the comment on the first file in "files changed" for more infos) 

(Don't trust my github name, it's Weide not Lyacs normally)
## Why / Balance

Adding these stamps encourga further lore-friendly (tg-lore mostly) shenanigans, gimmics, maps buildings, admemes, etc. Eventually I hope it will encourage better coders to develop a bit the content related to these factions

While it will more likely be used in MRP (hopefully), I hope to see them, and the lore around them, also in LRP

## Technical details

Added sprites, stamps entity, and stamps components based on [#668](https://github.com/Goob-Station/Goob-Station/pull/668)
## Media

cybersun stamp example
stamp-cybersun
paper_stamp-cybersun

ssc stamp example 
stamp-ssc
paper_stamp-ssc

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- add: Added a lot of stamps from NT corporates enemies as well as other lore related entities

### CODE REVIEW

**Comment by Lyacs** [comment]
I managed to finally setup the test environment. This PR still has two small issues.

One fixed and one I need to work on.

**Comment by Lyacs** [comment]
Nothing left to change on my end. I might add a few remaining stamps 'Gorlex', 'Tiger', 'Clown school', etc.  in the future, but not in this PR.

**Review by Piras314 — APPROVED** [review]

**Review by Piras314 — APPROVED** [review]
just to be sure, do these come from tgstation like the others? If they don't, you have to add their credits in the copyright section of the meta.json files

**Comment by Lyacs** [comment]
I created the sprites myself (colouring), using the base of the other stamps in the files. For the sprite that is shown on the paper it's 100% from my own creations. 

So should I add myself in the credits ?

**Comment by Aidenkrz** [comment]
> So should I add myself in the credits ?

Yes.
Also remove the UWU shit

**Review by Aidenkrz — CHANGES_REQUESTED** [review]
> So should I add myself in the credits ?

Yes
Also remove the UWU shit, we don't want that

**Comment by Piras314** [comment]
@Aidenkrz

File: Resources/Textures/Objects/Misc/bureaucracy.rsi/meta.json [hunk 1]
```diff
@@ -236,6 +236,33 @@
     },
     {
       "name": "paper_stamp-psychologist"
-    }
+    },
+      {
```
Comment (Aidenkrz) [inline Resources/Textures/Objects/Misc/bureaucracy.rsi/meta.json:240]: All of this is indented incorrectly, and there's 3 empty lines at the bottom for now reason.

**Review by Aidenkrz — CHANGES_REQUESTED** [review]
Mostly fine with this now, but none of these really spawn in anywhere, NTR should spawn with their stamp, and you should figure out ways for the rest of these to be obtainable, as admins can just make their own stamps, adding admin only ones is kind of useless and just bloat.

**Comment by Lyacs** [comment]
Closing it and making a brand new to save some

**Pull request closed without merging**